src/pages/Guilds/Guilds.tsx

```diff
@@ -47,20 +49,31 @@ const proposalsMock = [
 ];
 
 const GuildsPage: React.FC = () => {
+  // @TODO: parse guild contract address from where?
+  // @TODO: loading designs
+  // @TODO: error designs
+  const GUILDS_ADDRESS = '0x9cDC16b5f95229b856cBA5F38095fD8E00f8edeF';

```

Comment:
Can we move this to a constants file or a config? I think all chains will have the same contract address, but will need to confirm that.

Reply:
@rossneilson suggested using this from the URL, which is the proper way to do this. I think we can redirect from `/guilds` to `/guilds/0x9cDC16b5f95229b856cBA5F38095fD8E00f8edeF` for now?